inspirehep/modules/migrator/cli.py

```diff
@@ -134,6 +135,13 @@ def get_collection(marc_record):
                 return collection
         return 'HEP'
 
+    def fix_schema(json_record):
```

Comment:
Apparently, this code already exists at least in another place, like https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/authors/tasks.py#L65. Shall we move it somewhere else to make it reusable?

Reply:
There's also https://github.com/inspirehep/inspire-next/blob/a731e4f43bbded5cf5663e814f00c05e687c45aa/inspirehep/modules/workflows/tasks/upload.py#L90-L95. So it makes sense to pull this out into a separate function. What's a good place?